app/src/main/java/eu/tutorials/futureweatherforecast/activity/MainActivity.kt

```diff
@@ -51,88 +53,47 @@ class MainActivity : AppCompatActivity() {
->>>>>>> 584cea831f73f803de916498d486698d845a136e
             startActivity(intent)
         }
         bindind.manageCity.setOnClickListener {
             val intent = Intent(this@MainActivity, ManageCities::class.java)
             startActivity(intent)
         }
 
-        bindind.searchButton.setOnClickListener {
-            Constants.LOCATION = bindind.inputLocation.text.toString()
-            var locationinput: String = bindind.inputLocation.text.toString()
-            if (locationinput == "") {
-                getLocationWeatherDetails()
-            } else {
-                getLocationWeather()
-            }
-
-        }
-
-        mySwitch = bindind.nightMode as Switch
-        mySwitch.setOnClickListener {
-            if (mySwitch.isChecked) {
-                bindind.MainBG.setBackgroundResource(R.drawable.bgcloudynight)
-                bindind.nightMode.setTextColor(Color.WHITE)
-                bindind.tempText.setTextColor(Color.WHITE)
-                Toast.makeText(this, "Night mode is on", Toast.LENGTH_SHORT).show()
-            } else {
-                bindind.MainBG.setBackgroundResource(R.drawable.bgmorning)
-                bindind.nightMode.setTextColor(Color.BLACK)
-                bindind.tempText.setTextColor(Color.BLACK)
-                Toast.makeText(this, "Night mode is off", Toast.LENGTH_SHORT).show()
-            }
-        }
+//        bindind.searchButton.setOnClickListener {
+//            Constants.LOCATION = bindind.inputLocation.text.toString()
+//            val locationinput: String = bindind.inputLocation.text.toString()
+//            if (locationinput == "") {
+//                getLocationWeatherDetails()
+//            } else {
+//                getLocationWeather()
+//            }
+//        }
 
-        CtoF = bindind.CtoF
-        CtoF.setOnClickListener {
-            if (CtoF.isChecked) {
-                Constants.TemperatureScale = "F"
-                var locationinput: String = bindind.inputLocation.text.toString()
-                if (locationinput == "") {
-                    getLocationWeatherDetails()
-                } else {
-                    getLocationWeather()
-                }
-                bindind.CtoF.text = "Celcius"
-                   } else {
-                Constants.TemperatureScale = "C"
-                var locationinput: String = bindind.inputLocation.text.toString()
-                if (locationinput == "") {
-                    getLocationWeatherDetails()
-                } else {
-                    getLocationWeather()
-                }
-                bindind.CtoF.text = "Farenheit"
-                    }
-        }
         mFusedLocationClient = LocationServices.getFusedLocationProviderClient(this)
         mSharedPreferences = getSharedPreferences(Constants.PREFERENCE_NAME, Context.MODE_PRIVATE)
         //setupUI()
-        getLocationWeather()
+        var locationinput: String = Constants.LOCATION
+        if (locationinput == "") {
```

Comment:
the default value of the `locationInput` is in the constant, i think its good if you can update this condition to not equal to that constant value